tools/nuttx-gdbinit

```diff
@@ -455,7 +441,26 @@ define info_nxthreads
 end
 
 define nxcontinue
-  printf "nxcontinue\n"
   _restore_current_tcb
   continue
 end
+
+define info threads
+  info_nxthreads
```

Comment:
let's change info_nxthreads to info nxthreads

Reply:
Done